pkgs/top-level/emacs-packages.nix

```diff
@@ -507,6 +643,8 @@ rec {
     meta = { licence = gpl3Plus; };
   };
 
+  nyan-mode = callPackage ../applications/editors/emacs-modes/nyan-mode {};
```

Comment:
Because that derivation has a patch. If I were to inline it I would still need to place the patch somewhere (assuming one wouldn't want it inlined with some kind of `writeFile`).
AFAIKS the general policy with `<language>-packages.nix` is to inline only trivial builders.